documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -139,6 +139,24 @@ you will need the new `hibernate-search-elasticsearch` jar.
 ----
 ====
 
+If your Elasticsearch instance is hosted on Amazon Web Services,
+and if your instance is not public,
+you will need an additional dependency to handle authentication:
+`hibernate-search-elasticsearch-aws`.
+
+.Maven dependencies for Hibernate Search with AWS-hosted Elasticsearch
```

Comment:
Should we clarify these are not necessary when using our modules pack on WildFly ?